load/load.py

```diff
@@ -1,32 +1,34 @@
-import torch.multiprocessing as mp
 from sampling.sample_creator import SampleCreator
-
+from sampling.batch_creator import BatchCreator
+from multiprocessing import Event
 
 class DataLoader():
 
-  def __init__(self, data_store, batch_size=10):
-    """
-      Start the batchCreator and sampleCreator.
-      Read the memory config file, and create the right number
-      of processes.
-    """
-    self.data_store = data_store
-    self.batch_size = batch_size
-    
-    # Start separate processes for sample_creator(s)
-    sc = SampleCreator(self.data_store)
-    sc.start()
-    
-    # Start batch_creator(s)
+    def __init__(self, data_store, batch_size=10):
+        """
+          Start the batchCreator and sampleCreator.
+          Read the memory config file, and create the right number
+          of processes.
+        """
+        self.data_store = data_store
+        self.data_store.batch_size = batch_size
+        # Event to stop batch creator and sample creator
+        self.event = Event()
```

Comment:
Can you also add a function to set this event?

Reply:
Yup, added. It will be there in next push